pkg/repository/v1manifest/types.go

```diff
@@ -253,3 +255,28 @@ func (manifest *Snapshot) Filename() string {
 func (manifest *Timestamp) Filename() string {
 	return ManifestFilenameTimestamp
 }
+
+// HasNightly return true if the component has nightly version.
+func (manifest *Component) HasNightly() bool {
+	// TODO how to support nightly??
```

Comment:
Check whether does the versions list contains the nightly version?

Reply:
Platforms map[string]map[string]VersionItem contains no nightly now, i am not sure how it will be support